logging_exclusions.tf

```
@@ -0,0 +1,54 @@
+locals {
+  # Generate filter expressions for each user agent
+  json_payload_filters = [for agent in var.probe_user_agents : "jsonPayload.http_user_agent=~\"${agent}\""]
+  text_payload_filters = [for agent in var.probe_user_agents : "textPayload=~\"${agent}\""]
+
+  # Combine all filters into one string
+  probe_filter_expr = join(" OR ", concat(local.json_payload_filters, local.text_payload_filters))
+  probe_filter      = "resource.type=\"k8s_container\"\n(${local.probe_filter_expr})"
```

```suggestion
  probe_filter_expr = join(" OR ", concat(local.json_payload_filters, local.text_payload_filters))
  probe_filter      = "resource.type=\"k8s_container\" AND (${local.probe_filter_expr})"
```

I think that this could be more readable, what do you think?